src/components/data/StatCard.tsx

```diff
@@ -21,9 +23,10 @@ export default function StatCard({ value, label, className }: Props) {
         value
       )}
       {typeof label === "string" ? (
-        <div className="text-sm text-gray-50 sm:text-lg">{label}</div>
+        <div className="text-sm text-gray-50 sm:text-lg">{t(label)}</div>
```

Comment:
I think it shouldn't be responsible for translating. Let's keep the logic outside of this component